pnparser/pnparser.py

```diff
@@ -0,0 +1,191 @@
+                                       f"`{ctx.message.system_content}` executed at "
+                                       f"*{ctx.guild.name} - #{ctx.channel.name}:*\n"
+                                       f"https://discord.com/channels/{ctx.guild.id}/"
+                                       f"{ctx.channel.id}/{ctx.message.id}\n"
+                                       f"```fix\n{message}\n```"
+                                       "\n**Additional information**\n"
+                                       f"Patch notes: {patch_notes.patch_url}\n```python\n"
+                                       f"Cog version: {CURRENT_VERSION}\n"
+                                       f"Data Dragon version: {Dragon.current_version}\n```")
+        await ctx.send("Something went wrong and I could not parse.\n"
+                       "An automatic error report was generated and "
+                       "someone will look into it.")
+
+    def __validate_patch(self, patch_version: str) -> bool:
+        if Regex.search(r'^\s*[1-9]{1,2}(\.|,|-)[1-9]{1,2}\s*$', patch_version):
+            self.patch_version = patch_version.strip().replace(',', '.').replace('-', '.')
+            return True
+        return False
+
+    @commands.group()
+    async def pnparser(self, ctx: GuildContext) -> None:
+        """A League of Legends patch notes parser"""
+        pass
+
+    @pnparser.command()
+    async def version(self, ctx: GuildContext) -> None:
+        """Show package information"""
+        await ctx.send(f"Current version is {CURRENT_VERSION}.")
+
+    @pnparser.command()
+    async def report(self, ctx: GuildContext, *message: str) -> None:
+        """Report parser issues"""
+        await Tunnel.message_forwarder(destination=self.bug_fix_channel,
+                                       content="Patch notes parser error reported by "
+                                       f"**{ctx.author.display_name}** in "
+                                       f"*{ctx.guild.name} - #{ctx.channel.name}:*\n"
+                                       f"https://discord.com/channels/{ctx.guild.id}/"
+                                       f"{ctx.channel.id}/{ctx.message.id}\n{' '.join(message)}\n"
+                                       "\n**Additional information**\n"
+                                       f"Patch notes: {self.patch_notes.patch_url}\n```python\n"
+                                       f"Cog version: {CURRENT_VERSION}\n"
+                                       f"Data Dragon version: {Dragon.current_version}\n```")
+        await ctx.send("Thank you for reporting this issue.\n"
+                       "Someone will look into it and might get in touch "
+                       "to let you know when it is fixed.")
+
+    @pnparser.command()
+    async def reparse(self, ctx: GuildContext) -> None:
+        """Reparse the last parsed patch notes"""
+        if not self.patch_notes.patch_version:
```

Comment:
This should also be saved in a config object rather than just in memory so it persists across bot shutdowns

Reply:
Done, `[p]pnparser get patchversion` will tell you the current version once it has run at least once, otherwise it will return `None`.